esx_service/utils/auth_api.py

```diff
@@ -488,9 +488,26 @@ def vm_in_any_tenant(vms):
 
     return None
 
+
+def named_tenant(func):
+
+    def not_supported():
+        error_info = error_code.generate_error_info(ErrorCode.FEATURE_NOT_SUPPORT,
+                                                    auth_data_const.DEFAULT_TENANT)
+        return error_info
+
+    def wrap_function(name, vm_list):
```

Comment:
ok. Can you name it a bit better though, e.g. `check_args` ? wrap_function is too generic